app/src/pointing/input_split.c

```diff
@@ -56,8 +56,10 @@ int zmk_input_split_report_peripheral_event(uint8_t reg, uint8_t type, uint16_t
                  "Peripheral input splits need an `input` property set");                          \
     void split_input_handler_##n(struct input_event *evt) {                                        \
         for (size_t i = 0; i < ARRAY_SIZE(processors_##n); i++) {                                  \
-            zmk_input_processor_handle_event(processors_##n[i].dev, evt, processors_##n[i].param1, \
-                                             processors_##n[i].param2, NULL);                      \
+            int ret = zmk_input_processor_handle_event(processors_##n[i].dev, evt,                 \
+                                                       processors_##n[i].param1,                   \
+                                                       processors_##n[i].param2, NULL);            \
+            if (ret == ZMK_INPUT_PROC_CONTINUE) { continue; } else { return; }                     \
```

Comment:
The `continue;` seems a bit superfluous. How about:

```suggestion
            if (ret != ZMK_INPUT_PROC_CONTINUE) { return; }                     \
```

Reply:
Why not! Just updated.